src/components/popup-dialog/PopupDialog.tsx

```diff
@@ -23,8 +23,13 @@ const PopupDialog: FC<PopupDialogProps> = ({
 }) => {
   const { isMobile } = useBreakpoints()
 
-  const handleMouseOver = () => timerId && clearTimeout(timerId)
-  const handleMouseLeave = () => timerId && closeModalAfterDelay()
+  const handleMouseOver = () => {
+    if (timerId) clearTimeout(timerId)
+  }
+
+  const handleMouseLeave = () => {
+    if (timerId) closeModalAfterDelay()
```

Comment:
```suggestion
    if (timerId) closeModalAfterDelay(0)
```
or can we just use `closeModal`?

Reply:
Так ми можемо просто використати closeModalAfterDelay() без перевірки timerId. Це спростить код і зробить його читабельнішим